python/two-fer/two_fer.py

```diff
@@ -1,2 +1,6 @@
-def two_fer(name):
-    pass
+def two_fer(name = None):
```

Comment:
This is a good method of figuring out the default if you need to do some computation to arrive at it. For example None would be a good idea in a setup like:  
```
def func(x, n=None):
    if n is None:
        n = len(x) - 2
    ....
```
Yet, if you know upfront what your default will be, just go for:
`def two_fer(name='you'):`
This way you get rid of the if..